samples/src/Microsoft.Azure.IIoT.App/src/Pages/PublishedNodes.razor.cs

```diff
@@ -34,7 +34,7 @@ public partial class PublishedNodes {
             new PagedResult<ListNode>();
         private string _tableView = "visible";
         private string _tableEmpty = "displayNone";
-        private IAsyncDisposable _publishEvent { get; set; }
+        private IAsyncDisposable PublishEvent { get; set; }
```

Comment:
Why even use a property, why not just a variable?

Reply:
Agree. There are a number of properties that should be changed to variables. My suggestion would be to only use properties for data-binding, as unused properties are not shown in VS. I will create a work item for this.